libraries/ms-common/test/test_pwm.c

```diff
@@ -3,23 +3,26 @@
 #include "test_helpers.h"
 #include "unity.h"
 
+// TODO(ELEC-263): Consider adding functional or more rigorous testing.
+
 #define PERIOD_MS 100
 
 void setup_test(void) {}
 
 void teardown_test(void) {}
 
 void test_pwm_guards(void) {
-  TEST_ASSERT_EQUAL(STATUS_CODE_UNINITIALIZED, pwm_set_pulse(50));
-  TEST_ASSERT_EQUAL(STATUS_CODE_INVALID_ARGS, pwm_init(0));
-  TEST_ASSERT_OK(pwm_init(PERIOD_MS));
-  TEST_ASSERT_EQUAL(STATUS_CODE_INVALID_ARGS, pwm_set_dc(101));
-  TEST_ASSERT_EQUAL(STATUS_CODE_INVALID_ARGS, pwm_set_pulse(PERIOD_MS + 1));
+  TEST_ASSERT_EQUAL(STATUS_CODE_UNINITIALIZED, pwm_set_pulse(PWM_TIMER_3, 50));
```

Comment:
We should probably consider explicitly adding a platform-specific config of some kind. It would be helpful for tests and when we actually start using modules in application code.